### PR TITLE
[Token] Elevation (surface + shadow)

### PR DESCRIPTION
**Pull request by jeremie-lucca**
## Description

- Apply elevation tokens
- Deprecate current boxShadow variables

-----

image

image

-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2461/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2461/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/12493/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2461/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/12496/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2461/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/13244/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2461/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/13278/ui-diff)

**Comment by vvalentin-lucca** [comment]
I still see some `background-color: var(--colors-white-color);`
Shouldn’t they be replaced by `--pr-t-elevation-surface-default`?

**Comment by vvalentin-lucca** [comment]
`@include core.cssvars('commons-boxShadow', config.$boxShadows);` Shouldn't it be marked as depreciated?

**Comment by vvalentin-lucca** [comment]
I still find 4 calls to `var(--commons-boxShadow-`.

**Comment by vvalentin-lucca** [comment]
I still find 4 calls to `var(--commons-background-base)`.

File: packages/scss/src/commons/config.scss [hunk 1]
```diff
@@ -398,6 +399,20 @@ $boxShadows: (
 		'0 0 0 1px rgba(var(--colors-grey-900-rgb), 0.03), -2px 2px 8px rgba(var(--colors-grey-900-rgb), 0.04), -12px 6px 24px rgba(var(--colors-grey-900-rgb), 0.06)',
 );
 
+$elevation: (
+	surface: (
+		'sunken': 'var(--palettes-grey-50)',
+		'default': 'var(--palettes-grey-25)',
+		'raised': '#FFFFFF',
```
Comment (vvalentin-lucca) [inline packages/scss/src/commons/config.scss:406]: `var(--palettes-grey-text)`?

File: stories/qa/tokens/tokens.stories.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
+<h2>Elevation</h2>
+<h3>Surface</h3>
+<section class="contentSection">
+	<div class="demo-block">
+		<div class="demo-surface" style="background-color: var(--pr-t-elevation-surface-sunken);">Sunken</div>
+		<div class="demo-surface" style="background-color: var(--pr-t-elevation-surface-default);">Default</div>
+		<div class="demo-surface" style="background-color: var(--pr-t-elevation-surface-raised);">Raised</div>
+		<div class="demo-surface" style="background-color: var(--pr-t-elevation-surface-backdrop);">Backdrop</div>
```
Comment (vvalentin-lucca) [inline stories/qa/tokens/tokens.stories.html:8]: To put in first position?

**Comment by jeremie-lucca** [comment]
> I still see some `background-color: var(--colors-white-color);` Shouldn’t they be replaced by `--pr-t-elevation-surface-raised`?

I applied `--pr-t-elevation-surfaces` only on surfaces. `--colors-white-color` can also be used as text color or something else. Also, I considered a background as a surface only when a component can receive nested content. (eg, a card has a surface, a checkbox doesn't have.)
It might be refined later with extra design guidelines on tokens usage.

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2461/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/13396/ui-diff)

File: packages/scss/src/commons/config.scss (hunk 1 above)
Comment (jeremie-lucca) [reply packages/scss/src/commons/config.scss:474]: We can't use this one as it refers to a text color, but I can apply --colors-white-color.

**Review by vvalentin-lucca — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2461/storybook

**Review by vvalentin-lucca — APPROVED** [review]

**Review by Supamiu — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2461/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/13399/ui-diff)

**Comment by jeremie-lucca** [comment]
(Bypass due to old unsigned PR)

**Pull request merged**